src/main/java/nextstep/fp/StreamStudy.java

```diff
@@ -28,6 +29,13 @@ public static void printLongestWordTop100() throws IOException {
         List<String> words = Arrays.asList(contents.split("[\\P{L}]+"));
 
         // TODO 이 부분에 구현한다.
+        words.stream()
+                .filter(word -> word.length() > 12)
+                .map(String::toLowerCase)
+                .distinct()
+                .sorted(Comparator.comparing(String::length).reversed())
+                .limit(100)
+                .forEach(System.out::println);
```

Comment:
```suggestion
        String joined = words.stream()
                .filter(word -> word.length() > 12)
                .map(String::toLowerCase)
                .distinct()
                .sorted(Comparator.comparing(String::length).reversed())
                .limit(100)
                .collect(Collectors.joining(System.lineSeparator()));
        System.out.println(joined);
```
출력할 단어 수가 많아질 수록 System.out 콜이 늘어나게 되는 구조입니다.
이 코드뿐만 아니라 다른 코드에서도 I/O 를 줄이는 방향으로 나아가면 좋은 구조가 될거에요

Reply:
고민하다가 이번에는 단순히 Stream 사용해보는게 과제의 목적인 것 같아서 구현했었는데, 리팩토링해서 반영해볼게요!